eks/cluster-loader/configs/cluster-autoscaler/config.yaml

```diff
@@ -1,3 +1,5 @@
+# Adjust the following two for your cluster -
+### CAREFUL -- if (nodes*pods_per_node*client_qps % (pods_per_second*namespaces) != 0), test will fail. Ensure your parameters meet this condition.
```

Comment:
What do you mean by adjust the following two for your cluster? 

I'd also just use a single comment, unless you want to do something like:

##### README #####
# If (nodes*pods_per_node*client_qps % (pods_per_second*namespaces) != 0), test will fail. 
# This is a clusterloader2 restriction, so ensure your parameters meet this condition.

Reply:
Got it. I like this idea.